docs/pages/components/carousel/api/carousel.js

```diff
@@ -261,7 +261,7 @@ export default [
             },
             {
                 name: '<code>config</code>',
-                description: 'An object to pass all configs',
+                description: 'This property was removed, if you were using it, simply rename <code>:config</code> to <code>v-bind</code>',
```

Comment:
Since 0.9.0 will be considered a breaking release for Buefy, I think this part can be removed from the doc. Every breaking changes will need to be mentioned in the Changelog.

Reply:
Yep, there is an error thrown by the component if the prop is provided anyways, I'll remove it